build/dependencies.props

```diff
@@ -2,7 +2,8 @@
   <PropertyGroup>
     <AspNetCoreVersion>1.2.0-*</AspNetCoreVersion>
     <InternalAspNetCoreSdkVersion>2.1.1-*</InternalAspNetCoreSdkVersion>
-    <TestSdkVersion>15.3.0-*</TestSdkVersion>
-    <XunitVersion>2.3.0-beta2-*</XunitVersion>
+    <TestSdkVersion>15.3.0</TestSdkVersion>
```

Comment:
Why do we have exact versions for some of these while have `*`s for the others?

Reply:
@natemcmaster did we move away from using dependencies.props?

Reply:
Only if you want to use lineups. This repo doesn't really need lineups as its test projects use the latest RTM version of aspnetcore instead of the latest daily build.